src/data/mock.ts

```diff
@@ -0,0 +1,155 @@
+import { Semester, Course, CourseItem } from '.';
```

Comment:
Let's be more explicit with this import if we're able to change the name of the file where these are coming from.

`import <X> from '.';` doesn't tell me anything about where these imports are coming from unless there's only one other file in this folder. We can future proof this with `./<file_name>`

Reply:
As I mentioned before, importing the folder imports the index.js. This is a deterministic import, but I do agree that it isn't very descriptive none the less. I am going to change it.